ui/raidboss/data/05-shb/raid/e1s.js

```diff
@@ -3,6 +3,7 @@
 [{
   zoneRegex: {
     en: /^Eden's Gate: Resurrection \(Savage\)$/,
+    fr: /^L'Éveil d'Éden: Résurrection \(Sadique\)$/,
```

Comment:
Could you remove this line? zoneRegex doesn't behave like other translations.  It's only there for Chinese, Korean, and "English" (aka every language but Chinese and Korean).  This is because those versions use different ffxiv act plugins which translate the zone, whereas every other language uses the English zone names.

Reply:
I think this is also true for all of your other patches as well, sorry.

Reply:
oh sorry

Reply:
that's good. failed check, where is my error ? sorry.

Reply:
See https://github.com/quisquous/cactbot/blob/master/CONTRIBUTING.md#build-problems

Or run `npm run lint` and `npm run test` locally.

Reply:
i try, sorry.